docs/charts/bar.md

```diff
@@ -139,6 +139,16 @@ The bar chart defines the following configuration options. These options are mer
 | `gridLines.offsetGridLines` | `Boolean` | `true` | If true, the bars for a particular data point fall between the grid lines. The grid line will move to the left by one half of the tick interval. If false, the grid line will go right down the middle of the bars. [more...](#offsetgridlines)
 
 ### barThickness
+This option needs to be configured on x axis in scales property.
```

Comment:
It would be good to make this align with the docs for `offsetGridLines`. That means both that we should match the language (i.e. "This setting applies to the axis configuration.") and indentation (i.e. use 4-space indenting for the code sample)